test/e2e_test.go

```diff
@@ -1422,6 +1832,102 @@ func TestGenerate(t *testing.T) {
 	equals(ss.Optional["foo"], "bar", t)
 }
 
+func TestUploadDownload(t *testing.T) {
```

Comment:
Dup? https://github.com/sigstore/cosign/blob/7c20052077a81d667526af879ec40168899dde1f/test/e2e_attach_test.go#L310

Reply:
duplicate removed in 2f930d38